collect_app/src/main/res/layout/dialogbox_layout.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
If you are sure that we will need that in the future that's ok otherwise we should use LinearLayout.